.travis.yml

```diff
@@ -20,6 +20,7 @@ python:
 branches:
   only:
     - master
+    - requirements 
```

Comment:
you don't need to do this, travis is configured to automatically create builds for pull requests, please remove this line.

Reply:
Fixed it :)